### ISSUE TITLE
Spliced code, compilation step

### ISSUE THREAD

**Issue by devel-pa** [amark/gun#218]
Hi,
If you think that helps such a change: https://github.com/datuxer/gun
Best,
Paul

**Comment by amark** [comment]
@devel-pa I am very impressed! A lot of people just talk but very few actually dive in and do. I think you are an exemplar to others.

Personally, I do not like WebPack but I have told the community that if it is important to them that I would accept people who dive in and actually do the work to make it compatible. So I plan on **accepting this pull request given** a couple things that still need to be reviewed: If the output code still works/tested (it looks like it does from Travis?), there are no additional external dependencies added into the code (which it does not look like there is), and that the Babel compile targets do not break any legacy browsers that we support.

Overall, I'm very much impressed and want to praise you for having an attitude/approach that is respectful and involved. Rather than some people which I've seen be entitled and dismissive, you really shine a light on Open Source. Thank you!

Now, I do have some scary news. Most of GUN has been rewritten in the 0.5 branch for (1) less FRP usage and to support more procedural usage (2) performance (the new version can handle 30M+ ops/sec, while this current master/stable winds up choking under high ops/sec). So a lot of your work is going to get lost in the future when we switch between branches :( , however, if you still feel interested in being involved, I have designed a system that helps bridge the webpack/!webpack gap and would be very interested in hearing your thoughts on it. I will likely be more specific/particular about integration with 0.5, but hopefully you'll find the source code much more manageable (I'm impressed you got through the current version without asking me questions!). If you are interested, we should definitely chat about it. :)

It still might take me a bit of time to do the final review (all tests pass, no added dependencies, no breaking browser support - but most of these look like they are already done?) before we can get this merged. But lets get it merged. Thanks for being a great example to Open Source!

**Comment by devel-pa** [comment]
@amark What a speech ;)

Tested as much as I could. 

Probably incompatible with some old browser, I didn't care about IE8 and IE10 (discontinued) neither of IE9 (soon discontinued)

Added local storage as async and the possibility to attach a new one, API model is taken from `localForage` as it required minor changes.

I did that in the process of learning and exploring Gun, mostly because it's easier with small files.

Best,
Paul

**Comment by amark** [comment]
Thank you! I appreciate it. Again I really have to compliment you, I think you are setting a really good example for developers at large.

`localForage` actually sounds probably like a pretty good idea. As far as browser testing, you're gonna hate me, but with `0.5` we do some crazy compatibility and performance tests - check out this wiki: https://github.com/amark/gun/wiki/100000-ops-sec-in-IE6-on-2GB-Atom-CPU .

I look forward to talking! Hit me up with some times in the other thread.

**Comment by amark** [comment]
We finally got the `npm run unbuild` working and into master! So this should be compatible with webpack and others now. Thanks so much for helping me build it a few months ago, it went a long way! So I'm gonna close this issue now that it is merged. :)
